### ISSUE TITLE
Prevent downloading already downloaded datasets

### ISSUE THREAD

**Issue by quentinduche** [Inria-Empenn/shanoir_downloader#24]
We need this feature to prevent downloading already downloaded datasets.
I think this "filtering" should be performed at the dataset level before downloading datasets.

**Comment by alexpron** [comment]
I agree it would indeed be needed. At the moment it was not implemented as it represents a subsequent amount of work to be done in a proper way (see https://github.com/datalad/datalad-xnat as an example for xnat).
